tests/handlers/test_e2e_keys.py

```diff
@@ -148,7 +160,11 @@ def test_claim_one_time_key(self) -> None:
             )
         )
         self.assertDictEqual(
-            res, {"one_time_key_counts": {"alg1": 1, "signed_curve25519": 0}}
+            res,
+            {
+                "one_time_key_counts": {"alg1": 1, "signed_curve25519": 0},
+                "one_time_keys_count": {"alg1": 1, "signed_curve25519": 0},
+            },
```

Comment:
As per above---given that this affects clients rather than appservices I think we probably ought to leave this unchanged.